scripts/question_answering/commands/generate_commands.py

```diff
@@ -126,6 +126,14 @@ def gluon_en_cased_bert_base_v1_cfg():
     cfg.model_name = 'gluon_en_cased_bert_base_v1'
     return cfg
 
+def uncased_bert_wwm_large_cfg():
+    cfg = uncased_bert_base_cfg()
+    cfg.model_name = 'google_en_uncased_bert_wwm_large'
+    cfg.batch_size = 3
+    cfg.epochs = 2
+    cfg.max_seq_length = 384
```

Comment:
Let’s use 512 when releasing the scripts because it leads to better performance. Also, let’s update the fp16 training results in the markdown.